lib/verdict/experiment.rb

```diff
@@ -194,9 +194,9 @@ def remove_subject_assignment(subject)
     @storage.remove_assignment(self, subject)
   end
 
-  def switch(subject, context = nil)
+  def switch(subject, context = nil, qualifiers: [])
```

Comment:
Can we add documentation here for `qualifiers` and how they differ from the other kind of `@qualifiers`?

Reply:
Just added, does it make sense?

Reply:
🚢  Don't forget to squash.